docs/Custom-Brimcap-Config.md

```diff
@@ -160,22 +159,28 @@ analyzers:
   - cmd: /usr/local/bin/suricata-wrapper.sh
 ```
 
-A Brimcap analyzer has the following characteristics:
+To be invoked successfully by Brimcap, an analyzer process should have the
+following characteristics:
+
+1. It expects pcap input to be streamed to it via standard input (stdin).
+2. It is assumed that if stdin is closed on a process launched by the analyzer,
+that process will exit.
+3. The analyzer process must ensure that all processes that have inherited
+stdout/stderr have been killed before the analyzer process exits.
+4. It's expected to output log files that can be further processed as soon as
+lines are appended to them (i.e. `tail` could process them).
```

Comment:
Thanks @nwt. I'm happy to commit all your suggestions for this list of analyzer clarifications. I'd cc'ed @mattnibs  a couple times in these PR comments looking for clarifications on this same list since some of them were based on wording he provided, but I've not yet heard from him. Since I know you and he had a detailed Zoom session before his most recent round of Brimcap changes and you ultimately approved this PR, I'm assuming your wording improvements are based on a current understanding of the code, hence will merge.